src/main/java/exam2/lotto/LottoNumber.java

```diff
@@ -0,0 +1,94 @@
+package exam2.lotto;
+
+import java.util.Objects;
+
+public class LottoNumber implements Comparable<LottoNumber> {
+
+    /**
+     * 로또 시작 번호
+     */
+    public static final int LOTTO_START_NUMBER = 1;
+
+    /**
+     * 로또 마지막 번호
+     */
+    public static final int LOTTO_END_NUMBER = 45;
+
+    private int number;
+
+
+    /**
+     * 로또 번호 유효성 체크
+     *
+     * @param number
+     */
+    public LottoNumber(String number) {
+        this.number = checkValidRange(number);
```

Comment:
```suggestion
        this(checkValidRange(number));
```
다음과 같이 바꾸면 해당 생성자를 부 생성자로 사용함으로써 한군데에서 관리를 할 수 있을꺼 같아요!

Reply:
수정하였습니다!!